src/reconcile/rec_visibility.c

```diff
@@ -131,8 +131,14 @@ __rec_append_orig_value(
     if (WT_TIME_WINDOW_HAS_STOP(&unpack->tw)) {
         tombstone_globally_visible = __wt_txn_tw_stop_visible_all(session, &unpack->tw);
 
-        /* No need to append the tombstone if it is already in the update chain. */
-        if (oldest_upd->type != WT_UPDATE_TOMBSTONE) {
+        /*
+         * No need to append the tombstone if it is already in the update chain. it is an aborted
```

Comment:
What does this mean? It seems that you're saying that if there is a tombstone then it must be an aborted prepared update. I can't really make sense of that comment in relation to the code below it.

Reply:
Sorry, vscode truncated some words :(